tasking/pulp/tasking/base.py

```diff
@@ -350,7 +350,7 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
         :param einfo:   celery's ExceptionInfo instance, containing serialized traceback.
         :type einfo:    ???
         """
-        if isinstance(exc, PulpCodedException):
+        if isinstance(exc, PulpException):
```

Comment:
Do we want this if statment at all? I suspect celery will log the exception correctly in all situations right?